src/erlmld_batch_processor.erl

```diff
@@ -318,7 +318,7 @@ note_flush(State) ->
 elapsed_ms(When) ->
     trunc(timer:now_diff(
               os:timestamp(), When)
-              / 1.0e3).
+          / 1.0e3).
```

Comment:
Probably `1000` is more readable than `1.0e3`

Reply:
🤦